commands/clear_reports.js

```diff
@@ -0,0 +1,57 @@
+
+const Discord = require('discord.js');
+async function runCommand(server, rcon, msg, toClear, reason) {
+    if(!rcon.connected){
+        await msg.channel.send(`S${server} is not connected the bot.`)
+        return;
+    }
+    let res = await rcon.send(`/interface require("modules.control.reports").remove_all("${toClear}", "${msg.member.displayName}")`)
+    await msg.channel.send(`*Please make sure you have the correct username and a report was issued in #reports.* Server replies complete even if user does not exist. Server replied: ${res} By: ${msg.member.displayName} becasue: *${reason}*`);
+    console.log(`A user tried to Clear ${toClear}'s reports... \n    Server replied: ${res}    By: ${msg.member.displayName}/${msg.member.id} \n    Becasue: *${reason}*`);
+    let reportChan = msg.guild.channels.cache.get('368812365594230788'); // Reports channel is "368812365594230788" for exp // Reports Channel is "764881627893334047" for test server
+}
+
+
+module.exports = {
+    name: 'clear-reports',
+    aka: ['clearreports','cr','clear_reports','clear-report'],
+    description: 'Clear reports on ingame users (Admin/Mod only command)',
+    guildOnly: true,
+    args: true,
+    helpLevel: 'role.staff',
+    required_role: role.staff,
+    usage: ` <username> <reason>`,
+    async execute(msg, args, rcons, internal_error) {
+        const author = msg.author.username; //find author
```

Comment:
might wanna use displayname

Reply:
or do it with 24

Reply:
Do what with 24?

Reply:
just fix it :P

Reply:
```suggestion
        const author = msg.author.displayName; //find author
```
I don't even think I use it but I fixed it anyway ;)